src/evolution.cu

```diff
@@ -210,7 +208,7 @@ void apply_gauge(Grid &par, double2 *wfc, double2 *Ax, double2 *Ay,
         else{
             cMult<<<grid,threads>>>(wfc, (cufftDoubleComplex*) Ay, wfc);
         }
-        result = cufftExecZ2Z(plan_1d, wfc, wfc, CUFFT_INVERSE);
+        cufftExecZ2Z(plan_1d, wfc, wfc, CUFFT_INVERSE);
```

Comment:
This is true, but also applies to all cuda calls. I assumed that since we weren't handling errors on returns from `CudaMemcpy` and the like that we also shouldn't handle those from cuFFT calls.

If you think we should have an error handler I can put that in and add it to every cuda and cufft call.